src/app-service.ts

```diff
@@ -114,13 +114,9 @@ export class AppService extends EventEmitter {
             this.server = serverApp.listen(port, hostname, backlog, callback);
             return;
         }
-        return new Promise((resolve, reject) => {
-            this.server = serverApp.listen(port, hostname, backlog, (err: Error|null) => {
```

Comment:
The latest types for express removed the `err` parameter.

Reply:
That comes from Node's `server.listen()`.

Looks like nowadays you're either waiting from an `listening` or `error` event.
`server.on('listen', callback)` is the same as the callback parameter.

Reply:
I've tweaked it so we listen for error. It was a bit tedious because serverApp might either be a Server or an Application, but I think we're good now.